mesheryctl/internal/cli/root/root.go

```diff
@@ -115,6 +115,7 @@ func init() {
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
+	utils.CfgFile = cfgFile
```

Comment:
@navendu-pottekkat, @hexxdump, have we solidified plans to move us out of an 800 line utils file and toward a centralized struct that contains such references?

Reply:
There are some functions in utils.go that are expected to be moved into the mesheryctl config struct. Moved them should be a good step towards it.